src/modules/home/components/EventCarousel.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="relative rounded-2xl overflow-hidden h-40 w-120 z-20 shadow">
+    <event-carousel-slide
+      v-for="(event, slideIndex) in testEventProps"
+      :key="event.eventTitle"
+      :slideIndex="slideIndex"
+      :currentIndex="currentIndex"
+      :transitionAnimation="transitionAnimation"
+    >
+      <CardBanner class="w-full" :eventTitle="event.eventTitle" />
+    </event-carousel-slide>
+    <section class="carousel-misc absolute z-10">
+      <div
+        v-for="index in indexRange"
+        :key="index"
+        @click="changeSlide(index)"
+        class="w-1"
+        :class="{
+          'carousel-gap': index !== 3
+        }"
+      >
+        <div
+          class="carousel-dot bg-white"
+          :class="{
+            'h-3': index === currentIndex,

```

Comment:
I am not sure about vue js, but in react purgecss is broke when u do conditional renderering?

Please double check it.